helm/opendistro-es/values.yaml

```diff
@@ -39,9 +39,15 @@ kibana:
     kibana:
       enabled: false
       existingCertSecret:
+      existingCertSecretCertSubPath: kibana-crt.pem
+      existingCertSecretKeySubPath: kibana-key.pem
+      existingCertSecretRootCASubPath: kibana-root-ca.pem
     elasticsearch:
       enabled: false
       existingCertSecret:
+      existingCertSecretCertSubPath: elk-rest-crt.pem
+      existingCertSecretKeySubPath: elk-rest-key.pem
+      existingCertSecretRootCASubPath: elk-rest-root-ca.pem
```

Comment:
Use of the same name for the keys under kibana and elasticsearch is a bit confusing at first but I don't see a better alternative

Reply:
Well, kibana needs to use the same certificates as the rest layer of elasticsearch, so it needs to mount the _same_ secret as elasticsearch.

Reply:
Yeah, i don't see any other way. This is what they say in the documention so i would just stick with this :slightly_smiling_face: